GSoC-Contribution-Leaderboard/settings.py

```diff
@@ -164,3 +164,10 @@
 GITHUB_AUTH_TOKEN = config(
     'GITHUB_AUTH_TOKEN', default='GITHUB_TOKEN')
 
+# CELERY STUFF
+CELERY_BROKER_URL = 'redis://localhost:6379/'
```

Comment:
Please update the README.md as well to reflect all the changes in the steps that are needed to run the server now.